test/template_test.rb

```diff
@@ -15,6 +15,6 @@ def test_renders_basic_html
   def test_replaces_tag_content
     source = '<div data-prop="title">An Title</div>'
     output = render(source, 'title' => 'Ratchet')
-    assert_equal '<div>Ratchet</div>', output
+    assert_equal '<div data-prop="title">Ratchet</div>', output
```

Comment:
Cool: I'm reviewing commit-by-commit, and I was wondering if this property would stay in the output for future Jank use =]